src/ext/Util/ca/scaexec.cpp

```diff
@@ -1024,6 +1024,8 @@ extern "C" UINT __stdcall CreateUser(
         ::CoUninitialize();
     }
 
+    WcaLog(LOGMSG_VERBOSE, "iAttributes = %d, er = %d, hr = 0x%08x", iAttributes, er, hr);
```

Comment:
This looks like debug information being left in. Is that message text useful in the context of the log file or could more be added to make it useful? Or should it be removed?

Reply:
I'm adding additional logging to try to remote-debug the failure my change "introduced." I can't repro locally so I'm trying to narrow down possible causes. It will be removed when/if I fix the problem.